influxdb3_processing_engine/src/processing_engine_plugins.rs

```diff
@@ -0,0 +1,102 @@
+use influxdb3_wal::{PluginDefinition, TriggerSpecificationDefinition};
+use serde::de::Error;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Eq, PartialEq, Clone)]
+pub struct ProcessingEnginePlugin {
+    pub plugin_name: String,
+    pub code: String,
+    pub function_name: String,
```

Comment:
why have this and plugin name?

Reply:
This is the call site for the python code.

Reply:
Meaning that this is the Python function that Rust calls to hand over the rows to the Python plugin? If so, I think we should pick a static name and have that be part of the plugin authoring process. Your plugin always has that function defined. Convention over configuration 😄